opdracht1.js

```diff
@@ -0,0 +1,29 @@
+//Opdracht 1 - Array Methoden
+//Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+const tvNames = inventory.map((tvObject)=>{
+    return tvObject.type;
+});
+console.log(tvNames);
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const soldOut = inventory.filter((tvObject)=> {
+    if(tvObject.originalStock - tvObject.sold === 0){
+        return tvObject;
+    }
+});
```

Comment:
De filter functie verwacht een boolean als resultaat. Bij `true` zal de tv terugkomen, en bij `false` niet.
```suggestion
const soldOut = inventory.filter(tvObject=> tvObject.originalStock === tvObject.sold);
```